addons/web/static/src/js/views/form_common.js

```diff
@@ -1033,6 +1033,7 @@ var SelectCreatePopup = AbstractFormPopup.extend({
                 });
                 $cbutton = self.$buttonpane.find(".oe_selectcreatepopup-search-create");
                 $cbutton.click(function() {
+                    self.searchview.toggle_visibility(false);
```

Comment:
do't not change visibility on click of create button, it's should be in new_object method.
replace this.searchview.do_hide() at line 1160 (it hide only search $el not buttons) with self.searchview.toggle_visibility(false)